### DISCUSSION TITLE
Building in Fedora 40

### DISCUSSION THREAD

**Discussion by DieALot36T9** [quickemu-project/quickemu#1355] [Quickemu]
How do I build quickemu for fedora? I did not see any build instructions.
I would like to use it with the gui app.

**Comment by flexiondotorg** [comment]
Install requirements for Fedora are here:

- https://github.com/quickemu-project/quickemu/wiki/01-Installation#install-requirements-on-fedora

**Reply by DieALot36T9** [reply]
I have done that but I cannot use quickgui

**Reply by odd-git** [reply]
I have this issue too, but i can't run quickemu also from terminal. 
i install all the requirements
